rotkehlchen/chain/bitcoin/types.py

```diff
@@ -68,6 +68,19 @@ def deserialize_from_blockchain_info(
             direction=direction,
         )
 
+    @classmethod
+    def deserialize_from_haskoin(
+            cls,
+            data: dict[str, Any],
+            direction: BtcTxIODirection,
+    ) -> 'BtcTxIO':
```

Comment:
a question. don't you think it's better to move these functions to the `xxxManager` classes instead?